### PR TITLE
Add touch-action demo, based on Matt Gaunt's work.

### PR DESCRIPTION
**Pull request by sgomes**
Original demo: http://jsbin.com/batijohode/1/edit?html,css,output

@jeffposnick: PTAL!

FYI: @gauntface @PaulKinlan

### CODE REVIEW

File: touch-action/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+</section>
+
+<p>touch-action: pan-down</p>
+<section class="scroll-box" style="touch-action: pan-down;">
+  <div><div></div></div>
+  <div><div></div></div>
+  <div><div></div></div>
+</section>
+{% endcapture %}
+{% include output_helper.html initial_output_content=initial_output_content %}
```
Comment (jeffposnick) [inline touch-action/index.html]: You don't have to wrap this in the `output_helper.html` template. That template is useful when you're logging output or displaying status information, but if you just have some stuff in the DOM, it's not needed.

Comment (sgomes) [reply touch-action/index.html]: Good point, fixed.

**Comment by PaulKinlan** [comment]
@jeffposnick are we able to merge? I need to get this linked from the blog post.

**Comment by jeffposnick** [comment]
👍

**Pull request merged**